### PR TITLE
Add Loglevel option

### PR DESCRIPTION
**Pull request by matl33t**
Add flag for changing loglevel
Add `--debug` switch that enables GLI debug output

addresses https://github.com/lumoslabs/broadside/issues/2

### CODE REVIEW

File: lib/broadside/cli/global.rb [hunk 1]
```diff
@@ -34,6 +41,14 @@ def call_hook(type, command)
 
 pre do |global, command, options, args|
   Broadside.load_config(global[:config])
+
+  if global[:debug]
```
Comment (apurvis) [inline lib/broadside/cli/global.rb]: so you can only set log level if you set `--debug`?

**Comment by matl33t** [comment]
i'm thinking of changing `status` and `targets` to just do a `puts` to stdout instead, since they are commands that are really only run manually (rather than part of a script)

File: lib/broadside/cli/global.rb (hunk 1 above)
Comment (matl33t) [inline lib/broadside/cli/global.rb]: this is saying if `--debug` is set, ignore loglevel settings and force `Logger::DEBUG`

Comment (apurvis) [inline lib/broadside/cli/global.rb]: oops i got confused by the git view; thought this was all one block

**Comment by apurvis** [comment]
> i'm thinking of changing status and targets to just do a puts to stdout instead,

you mean just in case someone sets the log level too high and it doesn't work?

**Comment by matl33t** [comment]
right

**Comment by apurvis** [comment]
that makes sense

**Pull request merged**